src/Nethermind/Nethermind.Consensus/Processing/BlockCachePreWarmer.cs

```diff
@@ -24,7 +24,7 @@ public class BlockCachePreWarmer(ReadOnlyTxProcessingEnvFactory envFactory, ISpe
     private readonly ObjectPool<SystemTransaction> _systemTransactionPool = new DefaultObjectPool<SystemTransaction>(new DefaultPooledObjectPolicy<SystemTransaction>(), Environment.ProcessorCount);
     private readonly ILogger _logger = logManager.GetClassLogger<BlockCachePreWarmer>();
 
-    public Task PreWarmCaches(Block suggestedBlock, Hash256? parentStateRoot, CancellationToken cancellationToken = default)
+    public Task PreWarmCaches(Block suggestedBlock, Hash256? parentStateRoot, Address? beaconRootsAddress, CancellationToken cancellationToken = default)
```

Comment:
Would prefer to pass a list of AdditionalAddresses, not going into what those addresses are `params Span` would be most useful, but that is .net 9. This way it will be future proof.

Reply:
Could pass in a SystemTx AccessList maybe?

Reply:
Did AccesList and then can look at a .Combine for future SystemTxs